armi/utils/__init__.py

```diff
@@ -806,17 +806,39 @@ def merge(self, *otherDictionaries) -> None:
 
 def safeCopy(src: str, dst: str) -> None:
     """This copy overwrites ``shutil.copy`` and checks that copy operation is truly completed before continuing."""
+    # Convert files to OS-independence
+    src = os.path.abspath(src)
+    dst = os.path.abspath(dst)
```

Comment:
Hey @john-science I wanted to check with you on this choice. It's needed since the file paths were used to being handled via python and so going straight to the command line broke some file copies.

Reply:
Changing everything to absolute paths is generally a good idea and I like it.